src/Bicep.LangServer/Deploy/IDeploymentHelper.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.ResourceManager;
+using Bicep.LanguageServer.Handlers;
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public interface IDeploymentHelper
+    {
+        Task<BicepDeploymentStartResponse> StartDeploymentAsync(IDeploymentCollectionProvider deploymentCollectionProvider, ArmClient armClient, string documentPath, string template, string id, string scope, string location, string deploymentId, string portalUrl, string deploymentName, JsonElement parametersFileContents, IDeploymentOperationsCache deploymentOperationsCache);
```

Comment:
It's probably worth moving the function description comment from the `DeploymentHelper` class to be here instead as consumers will now be using the interface rather than the concrete class. You can annotate the concrete function with `<inheritdoc>` to indicate it should inherit the comment from the interface.

To give an example of how to do this:
- [INormalizedSchemaProvider](https://msazure.visualstudio.com/One/_git/AzureUX-Deployments?path=/src/Templates/Contracts/INormalizedSchemaProvider.cs&version=GBmaster&line=23&lineEnd=30&lineStartColumn=1&lineEndColumn=76&lineStyle=plain&_a=contents)
- [NormalizedSchemaProvider](https://msazure.visualstudio.com/One/_git/AzureUX-Deployments?path=/src/Engine/Host/Azure/Providers/NormalizedSchemaProvider.cs&version=GBmaster&line=47&lineEnd=47&lineStartColumn=9&lineEndColumn=26&lineStyle=plain&_a=contents)